addons/mail/__manifest__.py

```diff
@@ -233,6 +233,14 @@
             'mail/static/tests/qunit_mobile_suite_tests/**/*.js',
         ],
         # new bundles
+        'mail.assets_messaging_new': [
+            'mail/static/src/new/*.js',
+            'mail/static/src/new/**/*.js',
+            'mail/static/src/new/**/*.xml',
+            'mail/static/src/new/**/*.scss',
+            ('remove', 'mail/static/src/new/public/**/*'),
+            ('remove', 'mail/static/src/new/views/**/*'),
+        ],
```

Comment:
This bundle is useless. You can just import in `mail.assets_public` the files.
Also Keep it Dry: `src/new/**/` let you import everything in `/new`.